kitsune/wiki/views.py

```diff
@@ -1676,16 +1677,15 @@ def get_fallback_locale(doc, request):
     """Get best fallback local based on locale mapping"""
 
     # Get locales that the current article is translated into.
-    translated_locales = doc.translations.values_list("locale", flat=True).exclude(
-        current_revision=None
+    translated_locales = set(
```

Comment:
Why are you using `set` here?

Reply:
It's not a big speed improvement, but it does make the `x in translated_locales` checks more efficient (O(1) rather than O(n) where n is the number of translated locales).